FindMe/View Controllers/SignUpViewController.m

```diff
@@ -7,16 +7,220 @@
+
+@property (weak, nonatomic) IBOutlet UITextField *firstNameField;
+@property (weak, nonatomic) IBOutlet UITextField *lastNameField;
+@property (weak, nonatomic) IBOutlet UITextField *userNameField;
+@property (weak, nonatomic) IBOutlet UITextField *emailField;
+@property (weak, nonatomic) IBOutlet UITextField *passwordField;
+@property (weak, nonatomic) IBOutlet UIButton *registerButton;
+@property (strong,nonatomic) UITextField *activeField;
+@property (weak, nonatomic) IBOutlet UITextField *confirmPasswordField;
+@property (weak, nonatomic) IBOutlet UIImageView *profilePhoto;
+@property (weak, nonatomic) IBOutlet UIScrollView *scrollView;
+@property (weak, nonatomic) IBOutlet UILabel *confirmPasswordStatus;
+@property (weak, nonatomic) IBOutlet UILabel *emailStatus;
+@property (weak, nonatomic) IBOutlet UILabel *passwordStatus;
 
 @end
 
 @implementation SignUpViewController
 
 - (void)viewDidLoad
 {
+    CGFloat contentWidth = self.scrollView.bounds.size.width;
+    CGFloat contentHeight = self.scrollView.bounds.size.height;
+    self.scrollView.contentSize = CGSizeMake(contentWidth, contentHeight);
+    [self hideAlertLabels];
+    [self registerForKeyboardNotifications];
+    [self setTextFieldDelegates];
+    self.passwordField.secureTextEntry = YES;
+    self.confirmPasswordField.secureTextEntry = YES;
+    
+    [self createBottomBoarder:self.firstNameField];
+    [self createBottomBoarder:self.lastNameField];
+    [self createBottomBoarder:self.emailField];
+    [self createBottomBoarder:self.passwordField];
+    [self createBottomBoarder:self.userNameField];
+    [self createBottomBoarder:self.confirmPasswordField];
     [super viewDidLoad];
+    self.profilePhoto.layer.cornerRadius = 50;
+}
+
+- (IBAction)dismissKeyboard:(id)sender
+{
+    [self.view endEditing:YES];
+}
+
+- (void)hideAlertLabels
+{
+    self.passwordStatus.hidden =YES;
+    self.confirmPasswordStatus.hidden = YES;
+    self.emailStatus.hidden = YES;
+}
+
+- (IBAction)uploadProfilePhoto:(id)sender
+{
+    UIImagePickerController *imagePickerVC = [UIImagePickerController new];
+    imagePickerVC.delegate = self;
+    imagePickerVC.allowsEditing = YES;
+    
+    if ([UIImagePickerController isSourceTypeAvailable:UIImagePickerControllerSourceTypeCamera]) {
+        imagePickerVC.sourceType = UIImagePickerControllerSourceTypeCamera;
+    }
+    else {
+        NSLog(@"Camera 🚫 available so we will use photo library instead");
+        imagePickerVC.sourceType = UIImagePickerControllerSourceTypePhotoLibrary;
+    }
+    [self presentViewController:imagePickerVC animated:YES completion:nil];
+}
+
+- (void)setTextFieldDelegates
+{
+    self.userNameField.delegate = self;
+    self.lastNameField.delegate = self;
+    self.firstNameField.delegate = self;
+    self.emailField.delegate = self;
+    self.passwordField.delegate = self;
+    self.confirmPasswordField.delegate = self;
+}
+
+- (void)createBottomBoarder:(UITextField *)textField
```

Comment:
nit: `border`

I like helpers like this though. Almost warrants having a UITextField subclass. Like, `BottomBorderedTextField` or something.